core/src/packets/icmp/v6/ndp/router_solicit.rs

```diff
@@ -49,11 +49,14 @@ use std::fmt;
 ///                                 Otherwise, it *SHOULD* be included on link
 ///                                 layers that have addresses.
 ///
+/// The fields are accessible through [`Icmpv6<E, RouterSolicitation>`].
+///
 /// [`IETF RFC 4861`]: https://tools.ietf.org/html/rfc4861#section-4.1
+/// [`Icmpv6<E, RouterSolicitation>`]: Icmpv6
 #[derive(Clone, Copy, Debug, Default, Icmpv6Packet, SizeOf)]
 #[repr(C, packed)]
 pub struct RouterSolicitation {
-    reserved: u32,
+    _reserved: u32,
```

Comment:
minor, but maybe use the comment from below as a `//` internal comment for us?

Reply:
it's already in the main doc above.